jill/utils/defaults.py

```diff
@@ -28,15 +28,12 @@ def get_configfiles(filename):
 
 
 SOURCE_CONFIGFILE = get_configfiles("sources.json")
-RELEASE_CONFIGFILE = os.path.join(PKG_ROOT, "config", "releases.csv")
 GPG_PUBLIC_KEY_PATH = os.path.join(PKG_ROOT, ".gnupg", "juliareleases.asc")
+VERSIONS_URL = "https://julialang-s3.julialang.org/bin/versions.json"
```

Comment:
In China 😢 :

```text
requests.exceptions.SSLError: HTTPSConnectionPool(host='julialang-s3.julialang.org', port=443): Max retries exceeded with url: /bin/versions.json (Caused by SSLError(SSLError("bad handshake: SysCallError(54, 'ECONNRESET')")))
```

I'll add mirror support for this file as well after this PR...